docs-ui/components/panels.stories.js

```diff
@@ -90,6 +120,42 @@ export const _PanelTable = withInfo({
       <div>Panel Item</div>
       <div>Panel Item</div>
     </PanelTable>
+
+    <BulkController
+      pageIds={dummy.map(d => d.id)}
+      allIdsCount={23}
+      maxIdsCount={1000}
+      noticeColumns={3}
+    >
+      {({selectedIds, onPageIdsToggle, onIdToggle, isPageSelected, bulkNotice}) => (
+        <PanelTable
+          headers={[
```

Comment:
It's completely fine as you did but just a little suggestion. Maybe you could create a cons of type array, like:
`const headers = [....]` and reuse it here. Doing like this you could use `headers.length` in the noticeColumns

Reply:
Was considering it, but that variable would need to use `isPageSelected` and `onPageRowsToggle` render props and at the same time be used before those render props are available. So it just wasn't worth the hassle, especially just for a storybook demo.

Reply:
Oh yes I didn't notice that before